drivers/mpl3115a2/include/mpl3115a2_reg.h

```diff
@@ -18,8 +18,8 @@
  *
  */
 
-#ifndef __MPL3115A2_REG_H__
-#define __MPL3115A2_REG_H__
```

Comment:
Just to clarify this : I changed "__MPL3115A2_REG_H__" into "MPL3115A2_REG_H__" for example and you are saying that it should have been "MPL3115A2_REG_H" without the trailing double underscore correct?
I didn't change the trailing underscores since in the original PR they weren't mentioned and I checked some previous merges and they allowed trailing underscores.